### PR TITLE
Bug fix/satellite collection naming

### PR DESCRIPTION
**Pull request by Simran-B**
It should be spelled **SatelliteCollections** with upper-case S and C, without a space between both words according to @MrPieces.

This adjusts the spelling in code comments, error messages and DocuBlocks. Also fixed some other typos in errors.dat

Also needs to be fixed in documentation and on various pages of the website:
- https://www.arangodb.com/why-arangodb/arangodb-enterprise/
- https://www.arangodb.com/why-arangodb/arangodb-enterprise/arangodb-enterprise-satellite-collections/
- https://www.arangodb.com/master-services-agreement-inc-august-2019/
- https://www.arangodb.com/building-gdpr-compliant-applications/
- https://www.arangodb.com/?s=satellite+collections
- ...

### CODE REVIEW

File: js/apps/system/_admin/aardvark/APP/frontend/js/views/collectionsView.js [hunk 1]
```diff
@@ -646,9 +646,9 @@
               advancedTableContent.push(
                 window.modalView.createSelectEntry(
                   'is-satellite-collection',
-                  'Satellite collection',
+                  'SatelliteCollection',
```
Comment (KVS85) [inline js/apps/system/_admin/aardvark/APP/frontend/js/views/collectionsView.js]: Is it worth user-friendly?

Looking like this in UI:
image

File: js/apps/system/_admin/aardvark/APP/frontend/js/views/collectionsView.js [hunk 2]
```diff
@@ -646,9 +646,9 @@
...
                   '',
-                  'Create satellite collection? This will disable replication factor.',
+                  'Create SatelliteCollection? This will disable replication factor.',
```
Comment (KVS85) [inline js/apps/system/_admin/aardvark/APP/frontend/js/views/collectionsView.js]: Is it worth user-friendly?

Looking like this in UI:
image

**Comment by Simran-B** [comment]
@KVS85 The name is "SatelliteCollection", so spelling it differently is less user-friendly than a consistent spelling that matches up with what we advertise on the website (even though it is misspelled there too at the moment).

We also need to fix SmartJoins (SmartJOINS on the website?!) and check for other feature names as well.

Will require a rebuild of UI once more, and the error files errors.js and voc-errors.cpp need to be re-generated from errors.dat.

File: js/apps/system/_admin/aardvark/APP/frontend/js/views/collectionsView.js (hunk 2 above)
Comment (Simran-B) [reply js/apps/system/_admin/aardvark/APP/frontend/js/views/collectionsView.js:648]: Consistent spelling is important for the Enterprise Edition features, agreed by Jörg

**Comment by Simran-B** [comment]
https://jenkins01.arangodb.biz/view/PR/job/arangodb-matrix-pr/10147/

File: js/apps/system/_admin/aardvark/APP/react/build/asset-manifest.json [hunk 3]
```diff
@@ -1,26 +1,26 @@
...
-  "main.css": "static/css/main.7eea4f36.chunk.css",
-  "main.js": "static/js/main.5acb84ce.chunk.js",
-  "main.js.map": "static/js/main.5acb84ce.chunk.js.map",
+  "main.css": "static/css/main.6e703ac5.chunk.css",
+  "main.js": "static/js/main.7c5e95b6.chunk.js",
+  "main.js.map": "static/js/main.7c5e95b6.chunk.js.map",
   "static/css/1.0f119f1e.chunk.css": "static/css/1.0f119f1e.chunk.css",
-  "static/js/1.eb9b5434.chunk.js": "static/js/1.eb9b5434.chunk.js",
-  "static/js/1.eb9b5434.chunk.js.map": "static/js/1.eb9b5434.chunk.js.map",
+  "static/js/1.5455700b.chunk.js": "static/js/1.5455700b.chunk.js",
```
Comment (joerg84) [inline js/apps/system/_admin/aardvark/APP/react/build/asset-manifest.json:6]: Those changes are unrelated?

Comment (Simran-B) [reply js/apps/system/_admin/aardvark/APP/react/build/asset-manifest.json:6]: I believe the name is expected to change with every build (at least if something was modified), not sure if this done to avoid caching issues when served.

**Comment by Simran-B** [comment]
Adjusted tests to match changed error message wording:
https://jenkins01.arangodb.biz/view/PR/job/arangodb-matrix-pr/10153/

File: lib/Basics/errors.dat [hunk 4]
```diff
@@ -99,7 +99,7 @@ ERROR_ARANGO_SYNC_TIMEOUT,1111,"sync timeout","Will be raised when the server wa
 ERROR_ARANGO_CONFLICT,1200,"conflict","Will be raised when updating or deleting a document and a conflict has been detected."
 ERROR_ARANGO_DATADIR_INVALID,1201,"invalid database directory","Will be raised when a non-existing database directory was specified when starting the database."
 ERROR_ARANGO_DOCUMENT_NOT_FOUND,1202,"document not found","Will be raised when a document with a given identifier is unknown."
-ERROR_ARANGO_DATA_SOURCE_NOT_FOUND,1203,"collection or view not found","Will be raised when a collection with the given identifier or name is unknown."
+ERROR_ARANGO_DATA_SOURCE_NOT_FOUND,1203,"collection or View not found","Will be raised when a collection with the given identifier or name is unknown."
```
Comment (jsteemann) [inline lib/Basics/errors.dat:102]: This change may break clients or tests. I know it is just a text change, but unfortunately I have seen clients/tests not testing for the return codes but for return strings. This is a very common error code, so I have concerns someone tests on that exact string.

Comment (Simran-B) [reply lib/Basics/errors.dat:102]: Agreed, even some of our own tests do this.

BTW. The long description doesn't mention Views yet, will fix that but revert the change to the error text.

File: arangod/GeneralServer/AuthenticationFeature.cpp [hunk 5]
```diff
@@ -123,9 +123,10 @@ void AuthenticationFeature::collectOptions(std::shared_ptr<ProgramOptions> optio
       "--server.jwt-secret-folder",
       "folder containing one or more jwt secret files to use for jwt "
       "authentication. Files are sorted alphabetically: First secret "
-      "is used for signining + verifying jwt tokens. "
-      "(Enterprise only) The latter secrets are only used for verifying.",
-      new StringParameter(&_jwtSecretFolderProgramOption))
+      "is used for signing + verifying jwt tokens. The latter secrets "
```
Comment (Simran-B) [inline arangod/GeneralServer/AuthenticationFeature.cpp:126]:
```suggestion
      "is used for signing + verifying JWT tokens. The latter secrets "
```

File: lib/Basics/errors.dat [hunk 6]
```diff
@@ -30,7 +30,7 @@ ERROR_FILE_EXISTS,27,"file exists","Will be raised when a file already exists."
 ERROR_LOCKED,28,"locked","Will be raised when a resource or an operation is locked."
 ERROR_DEADLOCK,29,"deadlock detected","Will be raised when a deadlock is detected when accessing collections."
 ERROR_SHUTTING_DOWN,30,"shutdown in progress","Will be raised when a call cannot succeed because a server shutdown is already in progress."
-ERROR_ONLY_ENTERPRISE,31,"only enterprise version","Will be raised when an enterprise-feature is requested from the community edition."
+ERROR_ONLY_ENTERPRISE,31,"only Enterprise Edition","Will be raised when an Enterprise Edition feature is requested from the Community Edition."
```
Comment (Simran-B) [inline lib/Basics/errors.dat:33]:
```suggestion
ERROR_ONLY_ENTERPRISE,31,"only enterprise version","Will be raised when an Enterprise Edition feature is requested from the Community Edition."
```

File: lib/Basics/errors.dat (hunk 4 above)
Comment (Simran-B) [inline lib/Basics/errors.dat:102]:
```suggestion
ERROR_ARANGO_DATA_SOURCE_NOT_FOUND,1203,"collection or view not found","Will be raised when a collection or View with the given identifier or name is unknown."
```

File: lib/Basics/errors.dat [hunk 7]
```diff
@@ -169,48 +169,48 @@ ERROR_REPLICATION_SHARD_NONEMPTY,1417,"shard not empty","Will be raised when a s
 ## ArangoDB cluster errors
 ################################################################################
 
-ERROR_CLUSTER_SERVER_UNKNOWN,1449,"got a request from an unkown server","Will be raised on some occasions when one server gets a request from another, which has not (yet?) been made known via the agency."
+ERROR_CLUSTER_SERVER_UNKNOWN,1449,"got a request from an unknown server","Will be raised on some occasions when one server gets a request from another, which has not (yet?) been made known via the Agency."
```
Comment (Simran-B) [inline lib/Basics/errors.dat:172]:
```suggestion
ERROR_CLUSTER_SERVER_UNKNOWN,1449,"got a request from an unkown server","Will be raised on some occasions when one server gets a request from another, which has not (yet?) been made known via the Agency."
```

File: lib/Basics/errors.dat [hunk 8]
```diff
@@ -169,48 +169,48 @@ ERROR_REPLICATION_SHARD_NONEMPTY,1417,"shard not empty","Will be raised when a s
...
 ERROR_CLUSTER_MUST_NOT_CHANGE_SHARDING_ATTRIBUTES,1469,"must not change the value of a shard key attribute","Will be raised if there is an attempt to update the value of a shard attribute."
 ERROR_CLUSTER_UNSUPPORTED,1470,"unsupported operation or parameter","Will be raised when there is an attempt to carry out an operation that is not supported in the context of a sharded collection."
-ERROR_CLUSTER_ONLY_ON_COORDINATOR,1471,"this operation is only valid on a coordinator in a cluster","Will be raised if there is an attempt to run a coordinator-only operation on a different type of node."
-ERROR_CLUSTER_READING_PLAN_AGENCY,1472,"error reading Plan in agency","Will be raised if a coordinator or DBserver cannot read the Plan in the agency."
-ERROR_CLUSTER_COULD_NOT_TRUNCATE_COLLECTION,1473,"could not truncate collection","Will be raised if a coordinator cannot truncate all shards of a cluster collection."
+ERROR_CLUSTER_ONLY_ON_COORDINATOR,1471,"this operation is only valid on a Coordinator in a cluster","Will be raised if there is an attempt to run a Coordinator-only operation on a different type of node."
```
Comment (Simran-B) [inline lib/Basics/errors.dat:191]:
```suggestion
ERROR_CLUSTER_ONLY_ON_COORDINATOR,1471,"this operation is only valid on a coordinator in a cluster","Will be raised if there is an attempt to run a Coordinator-only operation on a different type of node."
```

File: lib/Basics/errors.dat [hunk 9]
```diff
@@ -169,48 +169,48 @@ ERROR_REPLICATION_SHARD_NONEMPTY,1417,"shard not empty","Will be raised when a s
...
+ERROR_CLUSTER_READING_PLAN_AGENCY,1472,"error reading Plan in Agency","Will be raised if a Coordinator or DB-Server cannot read the Plan in the Agency."
```
Comment (Simran-B) [inline lib/Basics/errors.dat:192]:
```suggestion
ERROR_CLUSTER_READING_PLAN_AGENCY,1472,"error reading Plan in agency","Will be raised if a Coordinator or DB-Server cannot read the Plan in the Agency."
```

File: lib/Basics/errors.dat [hunk 10]
```diff
@@ -169,48 +169,48 @@ ERROR_REPLICATION_SHARD_NONEMPTY,1417,"shard not empty","Will be raised when a s
...
-ERROR_CLUSTER_ONLY_ON_DBSERVER,1477,"this operation is only valid on a DBserver in a cluster","Will be raised if there is an attempt to run a DBserver-only operation on a different type of node."
-ERROR_CLUSTER_BACKEND_UNAVAILABLE,1478,"A cluster backend which was required for the operation could not be reached","Will be raised if a required db server can't be reached."
-ERROR_CLUSTER_AQL_COLLECTION_OUT_OF_SYNC,1481,"collection is out of sync","Will be raised if a collection needed during query execution is out of sync. This currently can only happen when using satellite collections"
-ERROR_CLUSTER_COULD_NOT_CREATE_INDEX_IN_PLAN,1482,"could not create index in plan","Will be raised when a coordinator in a cluster cannot create an entry for a new index in the Plan hierarchy in the agency."
-ERROR_CLUSTER_COULD_NOT_DROP_INDEX_IN_PLAN,1483,"could not drop index in plan","Will be raised when a coordinator in a cluster cannot remove an index from the Plan hierarchy in the agency."
+ERROR_CLUSTER_ONLY_ON_DBSERVER,1477,"this operation is only valid on a DB-Server in a cluster","Will be raised if there is an attempt to run a DB-Server-only operation on a different type of node."
```
Comment (Simran-B) [inline lib/Basics/errors.dat:195]:
```suggestion
ERROR_CLUSTER_ONLY_ON_DBSERVER,1477,"this operation is only valid on a DBserver in a cluster","Will be raised if there is an attempt to run a DB-Server-only operation on a different type of node."
```

File: lib/Basics/errors.dat [hunk 11]
```diff
@@ -169,48 +169,48 @@ ERROR_REPLICATION_SHARD_NONEMPTY,1417,"shard not empty","Will be raised when a s
...
 ERROR_CLUSTER_UNKNOWN_DISTRIBUTESHARDSLIKE,1486,"must not have a distributeShardsLike attribute pointing to an unknown collection","Will be raised if one tries to create a collection which points to an unknown collection in its distributeShardsLike attribute."
-ERROR_CLUSTER_INSUFFICIENT_DBSERVERS,1487,"the number of current dbservers is lower than the requested replicationFactor","Will be raised if one tries to create a collection with a replicationFactor greater than the available number of DBServers."
-ERROR_CLUSTER_COULD_NOT_DROP_FOLLOWER,1488,"a follower could not be dropped in agency","Will be raised if a follower that ought to be dropped could not be dropped in the agency (under Current)."
+ERROR_CLUSTER_INSUFFICIENT_DBSERVERS,1487,"the number of current DB-Servers is lower than the requested replicationFactor","Will be raised if one tries to create a collection with a replicationFactor greater than the available number of DB-Servers."
```
Comment (Simran-B) [inline lib/Basics/errors.dat:203]:
```suggestion
ERROR_CLUSTER_INSUFFICIENT_DBSERVERS,1487,"the number of current dbservers is lower than the requested replicationFactor","Will be raised if one tries to create a collection with a replicationFactor greater than the available number of DB-Servers."
```

File: lib/Basics/errors.dat [hunk 12]
```diff
@@ -169,48 +169,48 @@ ERROR_REPLICATION_SHARD_NONEMPTY,1417,"shard not empty","Will be raised when a s
...
+ERROR_CLUSTER_COULD_NOT_DROP_FOLLOWER,1488,"a follower could not be dropped in Agency","Will be raised if a follower that ought to be dropped could not be dropped in the Agency (under Current)."
```
Comment (Simran-B) [inline lib/Basics/errors.dat:204]:
```suggestion
ERROR_CLUSTER_COULD_NOT_DROP_FOLLOWER,1488,"a follower could not be dropped in agency","Will be raised if a follower that ought to be dropped could not be dropped in the Agency (under Current)."
```

File: lib/Basics/errors.dat [hunk 13]
```diff
@@ -169,48 +169,48 @@ ERROR_REPLICATION_SHARD_NONEMPTY,1417,"shard not empty","Will be raised when a s
...
 ERROR_CLUSTER_SHARD_LEADER_REFUSES_REPLICATION,1489,"a shard leader refuses to perform a replication operation","Will be raised if a replication operation is refused by a shard leader."
 ERROR_CLUSTER_SHARD_FOLLOWER_REFUSES_OPERATION,1490,"a shard follower refuses to perform an operation that is not a replication","Will be raised if a non-replication operation is refused by a shard follower."
 ERROR_CLUSTER_SHARD_LEADER_RESIGNED,1491,"a (former) shard leader refuses to perform an operation, because it has resigned in the meantime","Will be raised if a non-replication operation is refused by a former shard leader that has found out that it is no longer the leader."
-ERROR_CLUSTER_AGENCY_COMMUNICATION_FAILED,1492,"some agency operation failed","Will be raised if after various retries an agency operation could not be performed successfully."
+ERROR_CLUSTER_AGENCY_COMMUNICATION_FAILED,1492,"some Agency operation failed","Will be raised if after various retries an Agency operation could not be performed successfully."
```
Comment (Simran-B) [inline lib/Basics/errors.dat:208]:
```suggestion
ERROR_CLUSTER_AGENCY_COMMUNICATION_FAILED,1492,"some agency operation failed","Will be raised if after various retries an Agency operation could not be performed successfully."
```

File: lib/Basics/errors.dat [hunk 14]
```diff
@@ -169,48 +169,48 @@ ERROR_REPLICATION_SHARD_NONEMPTY,1417,"shard not empty","Will be raised when a s
...
 ERROR_CLUSTER_NOT_LEADER,1496,"not a leader","Will be raised when an operation is sent to a non-leading server."
-ERROR_CLUSTER_COULD_NOT_CREATE_VIEW_IN_PLAN,1497,"could not create view in plan","Will be raised when a coordinator in a cluster cannot create an entry for a new view in the Plan hierarchy in the agency."
-ERROR_CLUSTER_VIEW_ID_EXISTS,1498,"view ID already exists","Will be raised when a coordinator in a cluster tries to create a view and the view ID already exists."
-ERROR_CLUSTER_COULD_NOT_DROP_COLLECTION,1499,"could not drop collection in plan","Will be raised when a coordinator in a cluster cannot drop a collection entry in the Plan hierarchy in the agency."
+ERROR_CLUSTER_COULD_NOT_CREATE_VIEW_IN_PLAN,1497,"could not create View in plan","Will be raised when a Coordinator in a cluster cannot create an entry for a new View in the Plan hierarchy in the Agency."
+ERROR_CLUSTER_VIEW_ID_EXISTS,1498,"View ID already exists","Will be raised when a Coordinator in a cluster tries to create a View and the View ID already exists."
```
Comment (Simran-B) [inline lib/Basics/errors.dat:212]:
```suggestion
ERROR_CLUSTER_COULD_NOT_CREATE_VIEW_IN_PLAN,1497,"could not create view in plan","Will be raised when a Coordinator in a cluster cannot create an entry for a new View in the Plan hierarchy in the Agency."
ERROR_CLUSTER_VIEW_ID_EXISTS,1498,"view ID already exists","Will be raised when a Coordinator in a cluster tries to create a View and the View ID already exists."
```

File: lib/Basics/errors.dat [hunk 15]
```diff
@@ -326,7 +326,7 @@ ERROR_LDAP_CANNOT_START_TLS,1805,"cannot start a TLS LDAP session","can not star
 ERROR_LDAP_FOUND_NO_OBJECTS,1806,"LDAP didn't found any objects","LDAP didn't found any objects with the specified search query"
 ERROR_LDAP_NOT_ONE_USER_FOUND,1807,"LDAP found zero ore more than one user","LDAP found zero ore more than one user"
 ERROR_LDAP_USER_NOT_IDENTIFIED,1808,"LDAP found a user, but its not the desired one","LDAP found a user, but its not the desired one"
-ERROR_LDAP_INVALID_MODE,1820,"invalid ldap mode","cant distinguish a valid mode for provided ldap configuration"
+ERROR_LDAP_INVALID_MODE,1820,"invalid LDAP mode","cant distinguish a valid mode for provided LDAP configuration"
```
Comment (Simran-B) [inline lib/Basics/errors.dat:329]:
```suggestion
ERROR_LDAP_INVALID_MODE,1820,"invalid ldap mode","cant distinguish a valid mode for provided LDAP configuration"
```

File: lib/Basics/errors.dat [hunk 16]
```diff
@@ -435,18 +435,18 @@ ERROR_MODULE_SYNTAX_ERROR,3101,"syntax error in module","The module could not be
 ERROR_MODULE_FAILURE,3103,"failed to invoke module","Failed to invoke the module in its context."
 
 ################################################################################
-## Enterprise errors
+## Enterprise Edition errors
 ################################################################################
 
 ERROR_NO_SMART_COLLECTION,4000,"collection is not smart","The requested collection needs to be smart, but it ain't"
-ERROR_NO_SMART_GRAPH_ATTRIBUTE,4001,"smart graph attribute not given","The given document does not have the smart graph attribute set."
+ERROR_NO_SMART_GRAPH_ATTRIBUTE,4001,"SmartGraph attribute not given","The given document does not have the SmartGraph attribute set."
```
Comment (Simran-B) [inline lib/Basics/errors.dat:442]:
```suggestion
ERROR_NO_SMART_GRAPH_ATTRIBUTE,4001,"smart graph attribute not given","The given document does not have the SmartGraph attribute set."
```

File: lib/Basics/errors.dat [hunk 17]
```diff
@@ -435,18 +435,18 @@ ERROR_MODULE_SYNTAX_ERROR,3101,"syntax error in module","The module could not be
...
+ERROR_ILLEGAL_SMART_GRAPH_ATTRIBUTE,4004,"attribute cannot be used as SmartGraph attribute","The given smartGraph attribute is illegal and cannot be used for sharding. All system attributes are forbidden."
+ERROR_SMART_GRAPH_ATTRIBUTE_MISMATCH,4005,"SmartGraph attribute mismatch","The SmartGraph attribute of the given collection does not match the SmartGraph attribute of the graph."
+ERROR_INVALID_SMART_JOIN_ATTRIBUTE,4006,"invalid SmartJoin attribute declaration","Will be raised when the smartJoinAttribute declaration is invalid."
+ERROR_KEY_MUST_BE_PREFIXED_WITH_SMART_JOIN_ATTRIBUTE,4007,"shard key value must be prefixed with the value of the SmartJoin attribute","when using smartJoinAttribute for a collection, the shard key value must be prefixed with the value of the SmartJoin attribute."
+ERROR_NO_SMART_JOIN_ATTRIBUTE,4008,"SmartJoin attribute not given or invalid","The given document does not have the required SmartJoin attribute set or it has an invalid value."
```
Comment (Simran-B) [inline lib/Basics/errors.dat:449]:
```suggestion
ERROR_KEY_MUST_BE_PREFIXED_WITH_SMART_GRAPH_ATTRIBUTE,4003,"in smart vertex collections _key must be prefixed with the value of the smart graph attribute","In a smart vertex collection _key must be prefixed with the value of the SmartGraph attribute."
ERROR_ILLEGAL_SMART_GRAPH_ATTRIBUTE,4004,"attribute cannot be used as smart graph attribute","The given smartGraph attribute is illegal and cannot be used for sharding. All system attributes are forbidden."
ERROR_SMART_GRAPH_ATTRIBUTE_MISMATCH,4005,"smart graph attribute mismatch","The SmartGraph attribute of the given collection does not match the SmartGraph attribute of the graph."
ERROR_INVALID_SMART_JOIN_ATTRIBUTE,4006,"invalid smart join attribute declaration","Will be raised when the smartJoinAttribute declaration is invalid."
ERROR_KEY_MUST_BE_PREFIXED_WITH_SMART_JOIN_ATTRIBUTE,4007,"shard key value must be prefixed with the value of the smart join attribute","when using smartJoinAttribute for a collection, the shard key value must be prefixed with the value of the SmartJoin attribute."
ERROR_NO_SMART_JOIN_ATTRIBUTE,4008,"smart join attribute not given or invalid","The given document does not have the required SmartJoin attribute set or it has an invalid value."
```

File: lib/Basics/errors.dat [hunk 18]
```diff
@@ -455,29 +455,29 @@ ERROR_INVALID_DISJOINT_SMART_EDGE,4010,"non disjoint edge found","Will be raised
 ################################################################################
 
 ERROR_CLUSTER_REPAIRS_FAILED,5000,"error during cluster repairs","General error during cluster repairs"
-ERROR_CLUSTER_REPAIRS_NOT_ENOUGH_HEALTHY,5001,"not enough (healthy) db servers","Will be raised when, during repairDistributeShardsLike, there must be a free db server to move a shard, but there is no candidate or none is healthy."
+ERROR_CLUSTER_REPAIRS_NOT_ENOUGH_HEALTHY,5001,"not enough (healthy) DB-Servers","Will be raised when, during repairDistributeShardsLike, there must be a free DB-Server to move a shard, but there is no candidate or none is healthy."
```
Comment (Simran-B) [inline lib/Basics/errors.dat:458]:
```suggestion
ERROR_CLUSTER_REPAIRS_NOT_ENOUGH_HEALTHY,5001,"not enough (healthy) db servers","Will be raised when, during repairDistributeShardsLike, there must be a free DB-Server to move a shard, but there is no candidate or none is healthy."
```

File: lib/Basics/errors.dat [hunk 19]
```diff
@@ -455,29 +455,29 @@ ERROR_INVALID_DISJOINT_SMART_EDGE,4010,"non disjoint edge found","Will be raised
...
 ERROR_CLUSTER_REPAIRS_REPLICATION_FACTOR_VIOLATED,5002,"replication factor violated during cluster repairs","Will be raised on various inconsistencies regarding the replication factor"
-ERROR_CLUSTER_REPAIRS_NO_DBSERVERS,5003,"no dbservers during cluster repairs","Will be raised if a collection that is fixed has some shard without DB Servers"
+ERROR_CLUSTER_REPAIRS_NO_DBSERVERS,5003,"no DB-Servers during cluster repairs","Will be raised if a collection that is fixed has some shard without DB-Servers"
```
Comment (Simran-B) [inline lib/Basics/errors.dat:460]:
```suggestion
ERROR_CLUSTER_REPAIRS_NO_DBSERVERS,5003,"no dbservers during cluster repairs","Will be raised if a collection that is fixed has some shard without DB-Servers"
```

File: lib/Basics/errors.dat [hunk 20]
```diff
@@ -455,29 +455,29 @@ ERROR_INVALID_DISJOINT_SMART_EDGE,4010,"non disjoint edge found","Will be raised
...
-ERROR_CLUSTER_REPAIRS_JOB_DISAPPEARED,5009,"move shard job disappeared during cluster repairs","Will be raised if a move shard job in the agency cannot be found anymore before it finished"
-ERROR_CLUSTER_REPAIRS_OPERATION_FAILED,5010,"agency transaction failed during cluster repairs","Will be raised if an agency transaction failed during either sending or executing it."
+ERROR_CLUSTER_REPAIRS_MISMATCHING_SHARDS,5007,"mismatching shards during cluster repairs","Will be raised if in a collection and its distributeShardsLike prototype collection some shard and its prototype have an unequal number of DB-Servers"
+ERROR_CLUSTER_REPAIRS_JOB_FAILED,5008,"move shard job failed during cluster repairs","Will be raised if a move shard job in the Agency failed during cluster repairs"
+ERROR_CLUSTER_REPAIRS_JOB_DISAPPEARED,5009,"move shard job disappeared during cluster repairs","Will be raised if a move shard job in the Agency cannot be found anymore before it finished"
+ERROR_CLUSTER_REPAIRS_OPERATION_FAILED,5010,"Agency transaction failed during cluster repairs","Will be raised if an Agency transaction failed during either sending or executing it."
```
Comment (Simran-B) [inline lib/Basics/errors.dat:467]:
```suggestion
ERROR_CLUSTER_REPAIRS_OPERATION_FAILED,5010,"Agency transaction failed during cluster repairs","Will be raised if an agency transaction failed during either sending or executing it."
```

File: lib/Basics/errors.dat [hunk 21]
```diff
@@ -504,20 +504,20 @@ ERROR_NO_SUCH_ACTION,6004,"no such maintenance action","No such maintenance acti
...
+ERROR_BACKUP_TOPOLOGY,7003,"backup does not match this topology","The hot backup set cannot be restored on non matching cluster topology"
+ERROR_NO_SPACE_LEFT_ON_DEVICE,7004,"no space left on device","No space left on device"
+ERROR_FAILED_TO_UPLOAD_BACKUP,7005,"failed to upload hot backup set to remote target","Failed to upload hot backup set to remote target"
+ERROR_FAILED_TO_DOWNLOAD_BACKUP,7006,"failed to download hot backup set from remote source","Failed to download hot backup set from remote source"
+ERROR_NO_SUCH_HOT_BACKUP,7007,"no such hot backup set can be found","Cannot find a hot backup set with this Id"
+ERROR_REMOTE_REPOSITORY_CONFIG_BAD,7008,"remote hot backup repository configuration error","The configuration given for upload or download operation to/from remote hot backup repositories is wrong."
+ERROR_LOCAL_LOCK_FAILED,7009,"some DB-Servers cannot be reached for transaction locks","Some of the DB-Servers cannot be reached for transaction locks."
+ERROR_LOCAL_LOCK_RETRY,7010,"some DB-Servers cannot be reached for transaction locks","Some of the DB-Servers cannot be reached for transaction locks."
```
Comment (Simran-B) [inline lib/Basics/errors.dat:516]:
```suggestion
ERROR_REMOTE_REPOSITORY_CONFIG_BAD,7008,"remote hotback repository configuration error","The configuration given for upload or download operation to/from remote hot backup repositories is wrong."
ERROR_LOCAL_LOCK_FAILED,7009,"some db servers cannot be reached for transaction locks","Some of the DB-Servers cannot be reached for transaction locks."
ERROR_LOCAL_LOCK_RETRY,7010,"some db servers cannot be reached for transaction locks","Some of the DB-Servers cannot be reached for transaction locks."
```

File: lib/Basics/errors.dat [hunk 22]
```diff
@@ -504,20 +504,20 @@ ERROR_NO_SUCH_ACTION,6004,"no such maintenance action","No such maintenance acti
...
+ERROR_HOT_BACKUP_CONFLICT,7011,"hot backup conflict","Conflict of multiple hot backup processes."
+ERROR_HOT_BACKUP_DBSERVERS_AWOL,7012,"hot backup not all DB-Servers reachable","One or more DB-Servers could not be reached for hot backup inquiry"
```
Comment (Simran-B) [inline lib/Basics/errors.dat:518]:
```suggestion
ERROR_HOT_BACKUP_DBSERVERS_AWOL,7012,"hot backup not all db servers reachable","One or more DB-Servers could not be reached for hot backup inquiry"
```

File: lib/Basics/errors.dat [hunk 23]
```diff
@@ -504,20 +504,20 @@ ERROR_NO_SUCH_ACTION,6004,"no such maintenance action","No such maintenance acti
...
 
 ################################################################################
-## Plan analyzers errors
+## Plan Analyzers errors
 ################################################################################
-ERROR_CLUSTER_COULD_NOT_MODIFY_ANALYZERS_IN_PLAN, 7021, "analyzers in plan could not be modified", "Plan could not be modified while creating or deleting analyzers revision"
+ERROR_CLUSTER_COULD_NOT_MODIFY_ANALYZERS_IN_PLAN,7021,"Analyzers in plan could not be modified","Plan could not be modified while creating or deleting Analyzers revision"
```
Comment (Simran-B) [inline lib/Basics/errors.dat:523]:
```suggestion
ERROR_CLUSTER_COULD_NOT_MODIFY_ANALYZERS_IN_PLAN,7021,"analyzers in plan could not be modified","Plan could not be modified while creating or deleting Analyzers revision"
```

File: tests/Cluster/ClusterRepairsTest.cpp [hunk 24]
```diff
@@ -263,7 +263,7 @@ TEST_F(ClusterRepairsTestBrokenDistribution,
 
   ASSERT_EQ(collectionResult.errorNumber(), TRI_ERROR_CLUSTER_REPAIRS_NOT_ENOUGH_HEALTHY);
   ASSERT_TRUE(0 == strcmp(TRI_errno_string(collectionResult.errorNumber()),
-                          "not enough (healthy) db servers"));
+                          "not enough (healthy) DB-Servers"));
```
Comment (Simran-B) [inline tests/Cluster/ClusterRepairsTest.cpp:266]:
```suggestion
                          "not enough (healthy) db servers"));
```

File: tests/Cluster/ClusterRepairsTest.cpp [hunk 25]
```diff
@@ -286,7 +286,7 @@ TEST_F(ClusterRepairsTestBrokenDistribution,
...
```
Comment (Simran-B) [inline tests/Cluster/ClusterRepairsTest.cpp:289]:
```suggestion
                          "not enough (healthy) db servers"));
```

File: tests/rb/HttpInterface/api-general-graph-spec.rb [hunk 26]
```diff
@@ -1210,7 +1210,7 @@ def check400VertexUnused (doc)
             def check404CRUD (doc)
               check404(doc)
               doc.parsed_response['errorNum'].should eq(1203)
-              doc.parsed_response['errorMessage'].should start_with("collection or view not found: ")
+              doc.parsed_response['errorMessage'].should start_with("collection or View not found: ")
```
Comment (Simran-B) [inline tests/rb/HttpInterface/api-general-graph-spec.rb:1213]:
```suggestion
              doc.parsed_response['errorMessage'].should start_with("collection or view not found: ")
```

File: tests/rb/HttpInterface/api-general-graph-spec.rb [hunk 27]
```diff
@@ -1295,7 +1295,7 @@ def check404Collection (doc)
               doc.parsed_response['error'].should eq(true)
               doc.parsed_response['code'].should eq(404)
               doc.parsed_response['errorNum'].should eq(1203)
-              doc.parsed_response['errorMessage'].should include("collection or view not found")
+              doc.parsed_response['errorMessage'].should include("collection or View not found")
```
Comment (Simran-B) [inline tests/rb/HttpInterface/api-general-graph-spec.rb:1298]:
```suggestion
              doc.parsed_response['errorMessage'].should include("collection or view not found")
```

File: lib/Basics/errors.dat [hunk 28]
```diff
@@ -455,29 +455,29 @@ ERROR_INVALID_DISJOINT_SMART_EDGE,4010,"non disjoint edge found","Will be raised
...
-ERROR_CLUSTER_REPAIRS_JOB_DISAPPEARED,5009,"move shard job disappeared during cluster repairs","Will be raised if a move shard job in the agency cannot be found anymore before it finished"
-ERROR_CLUSTER_REPAIRS_OPERATION_FAILED,5010,"agency transaction failed during cluster repairs","Will be raised if an agency transaction failed during either sending or executing it."
+ERROR_CLUSTER_REPAIRS_MISMATCHING_SHARDS,5007,"mismatching shards during cluster repairs","Will be raised if in a collection and its distributeShardsLike prototype collection some shard and its prototype have an unequal number of DB-Servers"
+ERROR_CLUSTER_REPAIRS_JOB_FAILED,5008,"move shard job failed during cluster repairs","Will be raised if a move shard job in the Agency failed during cluster repairs"
+ERROR_CLUSTER_REPAIRS_JOB_DISAPPEARED,5009,"move shard job disappeared during cluster repairs","Will be raised if a move shard job in the Agency cannot be found anymore before it finished"
+ERROR_CLUSTER_REPAIRS_OPERATION_FAILED,5010,"Agency transaction failed during cluster repairs","Will be raised if an agency transaction failed during either sending or executing it."
```
Comment (Simran-B) [inline lib/Basics/errors.dat:467]:
```suggestion
ERROR_CLUSTER_REPAIRS_OPERATION_FAILED,5010,"agency transaction failed during cluster repairs","Will be raised if an agency transaction failed during either sending or executing it."
```

**Comment by jsteemann** [comment]
http://172.16.10.101:8080/view/PR/job/arangodb-matrix-pr/10192/
